src/components/Button/index.tsx

```diff
@@ -134,7 +134,7 @@ export const Button: FC<ButtonProps> = ({
           [iconSizeClasses[size]]: !!Icon,
         })}
       >
-        {Icon ? <Icon className="h-5 w-5" /> : children}
+        {Icon ? <Icon className={`h-5 w-5`} /> : children}
```

Comment:
Why this change?

Reply:
Ops! Probably some autofix of my IDE, but it doesn't make any sense.